examples/hello_env/BUILD

```diff
@@ -0,0 +1,26 @@
+load(
+    "@io_bazel_rules_rust//rust:rust.bzl",
```

Comment:
I guess we should migrate to `@rules_rust`, but separate PR (and not necessarily you :) sounds good.